selftests/test_curl_client.py

```diff
@@ -322,15 +321,18 @@ def test_data_posted(self):
     def test_parallel_mod_enabled(self):
         client = self.get_client("parallel")
         server = self.get_server("deproxy")
+        client.dump_headers = False
```

Comment:
Please add comment about `dump_headers` to code.

Reply:
Done. #486 will add more info to curl client.